src/services/__mocks__/partyService.ts

```diff
@@ -319,7 +308,19 @@ export const mockedParties: Array<Party> = [
     ],
     vatNumber: '111122211111',
     // Use case with no onboarded products
-    products: [],
+    products: [

```

Comment:
if you add a product, then remove the comment below

Reply:
removed comment